lab_1_classify_by_unigrams/main.py

```diff
@@ -34,7 +44,18 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if isinstance(tokens, list):
+        freq = {}
+        for i in tokens:
+            if isinstance(i, str) and len(i) == 1:
+                if i in freq:
+                    continue
+                if isinstance(i, str):
```

Comment:
How does it possible if `i` is not string in this line?

Reply:
done, it seems